### ISSUE TITLE
Che 6: ClassNotFoundException (LogstashEncoder) when starting Che

### ISSUE THREAD

**Issue by l0rd** [redhat-developer/rh-che#568]
The exception stack trace:

```
21:47:45,396 |-ERROR in ch.qos.logback.core.joran.action.NestedComplexPropertyIA - Could not create component [encoder] of type [net.logstash.logback.encoder.LogstashEncoder] java.lang.ClassNotFoundException: net.logstash.logback.encoder.LogstashEncoder
    at java.lang.ClassNotFoundException: net.logstash.logback.encoder.LogstashEncoder
    at     at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1291)
    at     at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1119)
    at     at ch.qos.logback.core.util.Loader.loadClass(Loader.java:120)
    at     at ch.qos.logback.core.joran.action.NestedComplexPropertyIA.begin(NestedComplexPropertyIA.java:102)
    at     at ch.qos.logback.core.joran.spi.Interpreter.callBeginAction(Interpreter.java:269)
    at     at ch.qos.logback.core.joran.spi.Interpreter.startElement(Interpreter.java:145)
    at     at ch.qos.logback.core.joran.spi.Interpreter.startElement(Interpreter.java:128)
    at     at ch.qos.logback.core.joran.spi.EventPlayer.play(EventPlayer.java:50)
    at     at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:165)
    at     at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:152)
    at     at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:110)
    at     at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:53)
    at     at ch.qos.logback.classic.util.ContextInitializer.configureByResource(ContextInitializer.java:75)
    at     at ch.qos.logback.classic.util.ContextInitializer.autoConfig(ContextInitializer.java:150)
    at     at org.slf4j.impl.StaticLoggerBinder.init(StaticLoggerBinder.java:84)
    at     at org.slf4j.impl.StaticLoggerBinder.<clinit>(StaticLoggerBinder.java:55)
    at     at org.slf4j.LoggerFactory.bind(LoggerFactory.java:150)
    at     at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:124)
    at     at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:412)
    at     at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:357)
    at     at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:383)
    at     at org.eclipse.che.inject.CheBootstrap.<clinit>(CheBootstrap.java:123)
    at     at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
    at     at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
    at     at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
    at     at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
    at     at org.apache.catalina.core.DefaultInstanceManager.newInstance(DefaultInstanceManager.java:120)
    at     at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4667)
    at     at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5207)
    at     at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
    at     at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:752)
    at     at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:728)
    at     at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:734)
    at     at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:988)
    at     at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1860)
    at     at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
    at     at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
    at     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
    at     at java.lang.Thread.run(Thread.java:748)
```

**Comment by skabashnyuk** [comment]
you do not have logstash-logback-encoder jar in your war file.

**Comment by pushpa1981** [comment]
Hi, @skabashnyuk , I am running into the same issue with springboot application running in IntelliJ.  how can I check whether the logstash-logback-encoder jar file fetched or not?  when this logstash-logback-encoder jar file fetched?

**Comment by skabashnyuk** [comment]
@pushpa1981 I didn't understand your question. Sorry. Can you rephrase?

**Comment by pushpa1981** [comment]
Thanks for the reply.
I was asking how to find out and check logstash-logback related libraries are pulled into the project?
I could able to figure out the issue . I have a network issue and the libraries didn't pulled. 
When I did Reimported all maven projects in Intellij IDE the libraries are pulled and the issue got resolved.
